e2e/MockStdinWatchPlugin.js

```diff
@@ -24,4 +24,13 @@ class MockStdinWatchPlugin {
     });
   }
 }
+
+/**
+ * Tried re-writing this in typescript but the tests just failed
+ *
+ * I've tried exporting them the following way
+ * 1. "module.exports = MockStdinWatchPlugin"
+ * 2. "exports = MockStdinWatchPlugin"
```

Comment:
what about `export = MockStdinWatchPlugin`? No "s"

Reply:
No dice. Doesn't work either.

Reply:
Ah, we don't transpile watch plugins. So this is actually expected - it needs to be JS